packages/nextra-theme/src/layout/MDXLayoutOutline.tsx

```diff
@@ -53,7 +66,7 @@ export const MDXLayoutOutline = () => {
                         transition: buildTransition('COLORS'),
                       }}
                     >
-                      {heading.value}
+                      {removeLinks(heading.value)}
```

Comment:
What does this do?

Reply:
when you have link in `heading`:

```mdx
## hello [bar](/my-link)
```

in toc it will be `<a />` inside `<a />`

<img width="1114" alt="image" src="https://github.com/user-attachments/assets/3e6d69c0-5f00-44d8-9d37-fd1ca0c15431">

```html
<li>
  <a class="..." href="...">
    hello <a href="/my-link">bar</a>
  </a>
</li>
```

so this function `removeLinks` remove wrapper links and converts them to 

```html
<li>
  <a class="..." href="...">
    hello bar
  </a>
</li>
```

Reply:
Nice! That makes sense.